### PR TITLE
Add nested expressions test + doc

### PR DESCRIPTION
**Pull request by mapedraza**
Should take the array order defined to apply expressions

### CODE REVIEW

File: test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js [hunk 1]
```diff
@@ -456,6 +456,31 @@ const iotAgentConfig = {
...
+                    type: 'Number',
+                    expression: 'v3*2',
+                    skipValue: 'notNull'
+                },
+                {
+                    name: 'value1',
+                    object_id: 'v1',
+                    type: 'Number',
+                    expression: 'v2*2',
+                    skipValue: 'notNull'
```
Comment (AlvaroVega) [inline test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js:483]: missed `},`

**Comment by fgalan** [comment]
Related issue? Or this is just a PR to harden our test base without any specific related issue?

**Comment by mapedraza** [comment]
> Related issue? Or this is just a PR to harden our test base without any specific related issue?

Added here: https://github.com/telefonicaid/iotagent-node-lib/issues/1493

**Review by AlvaroVega — APPROVED** [review]
LGTM

**Pull request merged**